src/controllers/beneficial.owner.type.controller.ts

```diff
@@ -11,10 +11,12 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     logger.debug(`GET ${config.BENEFICIAL_OWNER_TYPE_PAGE}`);
 
     const appData: ApplicationData = getApplicationData(req.session);
+    const statement = appData.beneficialOwnerStatement;
 
     return res.render(config.BENEFICIAL_OWNER_TYPE_PAGE, {
       backLinkUrl: config.BENEFICIAL_OWNER_STATEMENTS_URL,
-      ...appData.beneficialOwnerType
+      ...appData.beneficialOwnerType,
+      statement
```

Comment:
I think it would be better to check the statement types in the controller, to calculate the 2 flags, one to show the 'top half' (beneficial owners?) and one for the 'bottom half' (managers?). That would keep the logic in the template to a minimum, I think the templates should be as 'dumb' as possible. Passing a flag to display or not display a section should keep the template simple.